### PR TITLE
Feature/headeradjustments

### LINKED ISSUE #2063: Fix wrong color of header in safe area
On devices with using a safe area at the top, the background of the safe area is grey instead of white:

File

### PR DESCRIPTION
**Pull request by sagararyal**
Fixes #2063

Changes in this pull request:

- [x] fix issues related to small screen iPhone where tabs were hidden

### CODE REVIEW

**Comment by norbertschuler** [comment]
No solution compatible with other devices e.g. iPhone 6:

Simulator Screen Shot - iPhone 6 - 2020-03-23 at 19 42 34

**Comment by norbertschuler** [comment]
Works on Android:

Screenshot_1584967795

File: app/components/Competition/index.native.js [hunk 1]
```diff
@@ -122,7 +123,7 @@ class Competiton extends React.Component {
             navigation={this.props.navigation}
           />
           <Animated.View
-            style={{ marginTop: Platform.OS === 'ios' ? 24 : 56 }}
+            style={{ marginTop: Platform.OS === 'ios' ? height < 670 ? 56 : 26 : 56 }}
```
Comment (norbertschuler) [inline app/components/Competition/index.native.js]: This is a very device specific solution. Which iOS devices have you tested this with already?

Comment (harshvitra) [reply app/components/Competition/index.native.js]: I have referred to this design sheet for iOS - https://kapeli.com/cheat_sheets/iOS_Design.docset/Contents/Resources/Documents/index

The above will give you info in pixels you have to convert the pixels to points, you can do that here - https://www.ninjaunits.com/converters/pixels/pixels-points/

Rest is judgement on what looks good and what doesn't

**Comment by norbertschuler** [comment]
- [ ] This pull request has a side effect on the screens "Challenge friends" and "Edit profile" as following:

Simulator Screen Shot - iPhone 11 - 2020-03-24 at 09 23 05

Simulator Screen Shot - iPhone 11 - 2020-03-24 at 09 22 36

I don't know if our screens are also affected. Please test the app!

**Comment by norbertschuler** [comment]
- [ ] Now that the iPhone X and iPhone 11 look good, I see a grey bar at the top of the iPhone 6 simulator:

Simulator Screen Shot - iPhone 6 - 2020-03-24 at 09 45 49

**Comment by norbertschuler** [comment]
- [ ] On the iPhone 6 Plus Simulator (run `npm run ios -- --simulator="iPhone 6 Plus"`) the tab navigation under the headline is not visible (you can only see the green line selecting the tab entry):

Simulator Screen Shot - iPhone 6 Plus - 2020-03-24 at 09 55 56

File: app/components/Competition/index.native.js (hunk 1 above)
Comment (norbertschuler) [reply app/components/Competition/index.native.js]: Trying to build a manual solution for different devices sizes is difficult as e.g. now you missed big screens not having a notch: https://github.com/Plant-for-the-Planet-org/treecounter-app/pull/2060#issuecomment-602983070 - I thought the SafeAreaView was designed to solve this problem, so we do not have to implement our own solution?

**Pull request merged**